### DISCUSSION TITLE
v0.8.2

### DISCUSSION THREAD

**Discussion by Blinue** [Blinue/Magpie#358] [Announcements]
1. 感谢 @WHMHammer 将 README 和文档翻译为英语
2. 优化了 Graphics Capture 捕获模式的稳定性（感谢 @Pijulin）
3. 启用“在 3D 游戏中限制光标”时将无法使用多屏幕 #321（感谢 @mouarfff）
4. 修复鼠标加速关闭时无法正确调整光标速度的问题 #345（感谢 @ljx0772）
5. 优化了部分效果

<hr /><em>This discussion was created from the release <a href='https://github.com/Blinue/Magpie/releases/tag/v0.8.2'>v0.8.2</a>.</em>

**Comment by mouarfff** [comment]
非常感谢您 !

**Comment by Kuroonehalf** [comment]
This new version doesn't launch for me. Is anyone else getting this issue too?

**Reply by darklinkpower** [reply]
Neither for me. I opened an issue https://github.com/Blinue/Magpie/issues/359

**Comment by SpezFishz** [comment]
为什么我的帧率锁定在了60Fps 并且我是打开了无限制的

**Reply by Blinue** [reply]
Graphics Capture 的特色，更换捕获模式即可
